src/globalComponents/userAPIIntegration.js

```diff
@@ -1,24 +1,51 @@
 //@flow strict
 
 import Cookies from 'universal-cookie';
+import getLoginToken from '../globalComponents/getLoginToken.js';
 
 /*
 	This function has no input
 	This function returns a json object of the user
 */
-function getUser() : Promise<Response> {
+function getUser(): Promise<Response> {
 	const cookies = new Cookies();
-	const token = cookies.get('token');
 	const responsePromise: Promise<Response> = fetch('/api/user/getUser/', {
 		method: 'GET',
 		headers: {
 			'Access-Control-Allow-Origin': '*',
 			'Content-Type': 'application/json',
 			'Access-Control-Allow-Credentials': 'true',
-			'x-auth-token': token
+			'x-auth-token': cookies.get('token'),
 		},
 	});
 	return responsePromise;
 }
 
-export {getUser};
+// Sets the user's wager according to the number passed to it.
+// Returns a boolean indicating success (true) or failure (false). Maybe change to string to return the error.
+function setWager(wager: number, onLoad:(response: boolean) => void) {
```

Comment:
I would rename this 'loadSuccessful' or something because I think that makes more sense than 'response'. It isn't clear whether response being true means there was an error or something went wrong